app/components/display/search_result_component.rb

```diff
@@ -53,20 +57,30 @@ def mappings_button
 
   def visualize_button
       link_to_modal(nil, "/ajax/biomixer/?ontology=#{@ontology_acronym}&conceptid=#{@uri}", data: { show_modal_title_value: @title, show_modal_size_value: 'modal-xl' }) do
-        content_tag(:div, class: 'button') do
+        content_tag(:div, class: 'button', style: @portal_color ? "background-color: #{@portal_light_color} !important" : '') do
```

Comment:
this change is not need the visualize_button is not showed if the concept is external